.github/workflows/test-snap-can-build.yml

```diff
@@ -0,0 +1,25 @@
+name: 🧪 Test snap can be built on x86_64
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]    
```

Comment:
Is this needed?

Reply:
Yes, otherwise you get a deprecation warning from GH.

Reply:
Also, I think it's fine to omit this from the PR if I'm managing the snap.